src/Zynga/Framework/PgData/V1/PgModel/SqlGenerator.hh

```diff
@@ -15,6 +15,8 @@ use \Exception;
 
 class SqlGenerator {
 
+  const string SKIP_FIELD_PREFIX = '_ignore_';
```

Comment:
[question] - I see what you doing here, do you mind to provide more details on it? why and how we use this through PgModels?

Reply:
Good question. We need to be able to mark fields that we don't want translated into SQL statements but still want them to be set in non-sql formats (i.e. the cached JSON object) such as the tombstone field. This is to avoid having to modify all existing and future SQL tables to include this field.